modules/RTC/RTCUtils.js

```diff
@@ -377,7 +114,7 @@ function getConstraints(um, options = {}) {
  * @private
  * @returns {Object}
  */
-function newGetConstraints(um = [], options = {}) {
+function getConstraints(um = [], options = {}) {
```

Comment:
```
  if (options.resolution) {
            if (Resolutions[options.resolution]) {
                constraints.video.width = {
                    ideal: Resolutions[options.resolution].width
                };
                constraints.video.height = {
                    ideal: Resolutions[options.resolution].height
                };
            }
        }
```
I agree that we should bring back `config.resolution`, and that it should overwrite the video constraints here.